tests/unit/components/HdLazyImage.spec.js

```diff
@@ -0,0 +1,26 @@
+import { mount } from '@vue/test-utils';
+import HdLazyImage from '@/components/HdLazyImage.vue';
+import ITEMS from '@/stories/mocks/GALLERY_ITEMS';
+
+describe('HdLazyImage', () => {
+  let wrapper;
+  const sampleImage = ITEMS[2];
+
+  beforeEach(() => {
+    wrapper = mount(HdLazyImage, {
+      propsData: {
+        src: sampleImage.image,
+        srcSmall: sampleImage.thumbnail,
+        background: false,
+      },
+    });
+  });
+
+  it('renders component', () => {
+    expect(wrapper.html()).toMatchSnapshot();
+  });
+
+  it('It fist display the thumbnail/small image', () => {
+    expect(wrapper.attributes().src).toBe(sampleImage.thumbnail);
+  });
```

Comment:
Aren't those tests similar? I think you can condense both in just one test: snapshot + src.

Reply:
Yes. I could actually just keep the toMatchSnapshot one.